### PR TITLE
Fix deployment readme

### PR DESCRIPTION
**Pull request by saad-ali**
Follow-up PR will add example YAML

### CODE REVIEW

File: deploy/kubernetes/README.md [hunk 1]
```diff
@@ -1,18 +1,37 @@
-1. One-time per project: Create GCP service account for CSI driver
-    1. Export environment variables for location for service account private key file and name of the service account
+1. [One-time per project] Create GCP service account credential for CSI Driver and download JSON:
+    1. Go to https://pantheon.corp.google.com/apis/credentials
+    2. Select `Create credentials` -> ~Service account key`
+    3. For `Service Account` select `Compute Engine Default service account`
+    4. Select `JSON`
+    5. Click `Create` to download JSON for the private key to your local machine.
```
Comment (msau42) [inline deploy/kubernetes/README.md]: You shouldn't have to download keys from pantheon. The script should already create the service account and key.

File: deploy/kubernetes/README.md [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
+    6. Rename the downloaded JSON file to `cloud-sa.json`
+    6. Export environment variables for 1) location of service account private key JSON file, 2) name you want of service account to be created by `setup_project.sh`, and 3) GCP project:
     ```
-    $ export SA_FILE=~/.../cloud-sa.json
-    $ export GCEPD_SA_NAME=sample-service-account
+    $ export SA_FILE=~/Downloads/cloud-sa.json
```
Comment (davidz627) [inline deploy/kubernetes/README.md]: Could be any directory. Need to emphasise that it MUST be called "cloud-sa.json" though

File: deploy/kubernetes/README.md [hunk 3]
```diff
@@ -1,18 +1,37 @@
...
 ```
-3. Create example PVC and Pod
-```
-$ kubectl apply -f ./examples/demo-pod.yaml
-```
\ No newline at end of file
+3. Create example workload to verify
+    1. Deploy yaml for PVC and Pod
+    ```
+    $ kubectl apply -f ./examples/demo.yaml
```
Comment (davidz627) [inline deploy/kubernetes/README.md]: `./examples/kubernetes/demo.yaml` ?

Also we can take the demopod from `./deploy/kubernetes/demo-pod.yaml`

File: deploy/kubernetes/README.md (hunk 1 above)
Comment (saad-ali) [reply deploy/kubernetes/README.md]: @davidz627 could you clarify? Are these steps to create and download the key required? If not, is the target JSON path something that will be created?

Comment (davidz627) [reply deploy/kubernetes/README.md]: there is a script that does substeps 1-6, it is called `setup_project.sh`. As long as you set all the right env variables the script will handle creating service account, binding roles, and downloading the json key to the provided directory (`SA_FILE`)

**Comment by davidz627** [comment]
Incorporated into #86 as part of a larger documentation fix

**Pull request closed without merging**